BiasAndFairnessModule/run_full_evaluation.py

```
@@ -0,0 +1,345 @@
+import pandas as pd
+import numpy as np
+import json
+
+# Add the src directory to Python path
+current_dir = Path(__file__).parent
+src_dir = current_dir / "src"
+sys.path.insert(0, str(src_dir))
+
+# Import directly from the eval_engine directory
+from src.eval_engine.metrics import (
+    demographic_parity, equalized_odds, equalized_opportunity, predictive_equality,
```

Ruff (0.12.2)

`src.eval_engine.metrics.demographic_parity` imported but unused

Remove unused import

(F401)